.babelrc

```diff
@@ -0,0 +1 @@
+{ "stage": 1 }
```

Comment:
Sorry, the repo is using `babelify` version 6.1.2, which uses `babel-core` version 5: https://github.com/babel/babelify/blob/v6.1.2/package.json

Anyway, upgrading to babel6 is something that we should do, but it's out-of-scope for this PR.